src/temp/Explore/CustomSwitch.tsx

```diff
@@ -0,0 +1,57 @@
+import { Switch, styled } from '@mui/material';
+import theme from '../../theme/themeProperties';
+const {
+  primaryColorNew,
+  restorationToggleColorNew,
+  currentForestSwitchTrackColorNew,
+  restorationSwitchTrackColorNew,
+  deforestrationSwitchTrackColorNew,
+} = theme;
+interface SwitchProps {
```

Comment:
Everything looks fine, just a minute change. Could you changed the naming to `currentForestSwitch`, `restorationSwitch` and `deforestationSwitch` instead?